### ISSUE TITLE
replacement tool not working or am i idiot

### ISSUE THREAD

**Issue by thestuffyousentme** [Sadret/openrct2-scenery-manager#45]
For what I understand, replacement tool works as follows:

**1.** pick or select object to be replaced .
options are type1 (class)+type2(variant)+type3 (this would be an addition like a lamp) 

     e.g. footpath+green+bin

**2.** pick or select desired replacement for object **1**
options are the same as above (type1+type2+type3) with the addition that types 2 and 3 can remain untouched (keep), and type 3 can also be eliminated (none)

     e.g. footpath+green+none

this should result in the first object replaced by the second, wich in the given example, would be the same except for the none tag being applied wichh should remove the bin.

Am I wrong presuming how's this suppossed to work or why doesnt it work ?

thanks

**Comment by Sadret** [comment]
You are right, this seems to be a bug. Is the issue only with footpath additions?

**Comment by thestuffyousentme** [comment]
> You are right, this seems to be a bug. Is the issue only with footpath additions?

Hello! Thanks for answering.

I tried the other options, small scenery object, large scenery object and walls. it seems to work flawlessly in the three of them.

However with footpaths I only get to change the surface object, properly the footpath, and the railings, but there is no change in additions, e.g. be it replace a bin with a lamp or replace bin with none.

However, the game still charges money for the action, although there is no change in the tile properties. Also, the plug in reports that x objects have been correctly replaced with 0 errors.

Also, I captured a video:

https://streamable.com/exyaof

Thank you!
